webpack.config.js

```diff
@@ -69,6 +70,9 @@ module.exports = {
   devtool: isProduction ? 'source-map' : 'cheap-module-eval-source-map',
   plugins: (() => {
     let plugins = [
+      new FlowWebpackPlugin({
```

Comment:
Anecdotal testing shows that the compile time doubles when making changes and saving files when this plugin is enabled.  I'd really rather not increase reload time. Perhaps this should be behind an opt-in flag, similar to `build:analyze`.  Alternatively, consider researching if there is flow integration for your editor.

**without plugin:** 1914ms, 1862ms
**with plugin:** 4090ms, 4633ms, 4861ms